packages/headless/src/features/search-parameters/search-parameter-serializer.ts

```diff
@@ -30,7 +30,7 @@ function serializePair(pair: [string, unknown]) {
     return '';
   }
 
-  if (key === 'f' || key === 'cf') {
+  if (key === 'f' || key === 'cf' || key === 'sf') {
     return isFacetObject(val) ? serializeFacets(key, val) : '';
```

Comment:
I was thinking about renaming functions like `isFacetObject` and `serializeFacets` to something more general that includes `sf`. I couldn't find a nice name though ...

The most accurate would be something like `isStringStringArrayRecord`, which could be shortened to `isStringArrayRecord` since record keys are always strings.

Another idea is `isFacetLike` but I feel it is too vague/unclear. Not a fan,